opencsp/app/sofast/SofastGUI.py

```diff
@@ -53,7 +57,25 @@ def __init__(self) -> 'SofastGUI':
         # Run window infinitely
         self.root.mainloop()
 
-    def _get_frame(self) -> np.ndarray:
+    def _load_system_fringe_elements(self) -> bool:
+        """Loads the system instance, as appropriate.
+
+        Checks if System object can be instantiated (if both ImageAcquisition and ImageProjection classes are loaded)
+        """
+        camera = ImageAcquisitionAbstract.instance()
+        projector = ImageProjection.instance()
+        if camera is not None and projector is not None:
+            # Check that we don't already have a fringe instance
+            if self.sys_fringe is not None:
+                return True
+            # Create the new instance
+            self.sys_fringe = ssf.SystemSofastFringe(projector, camera)
```

Comment:
Looks like you edited SystemSofastFringe to only take "projector" as an input, not both projector and camera.

Reply:
So I did. Fixed!